src/main/proto/wfa/measurement/api/v2alpha/requisition.proto

```diff
@@ -58,6 +59,22 @@ message Requisition {
   // `DataProviderEntry` in `measurement`.  Required. Immutable.
   bytes encrypted_requisition_spec = 6;
 
+  message DuchyEntry {
+    // Resource key of the `Duchy` that this entry is for. Required. Immutable.
+    Duchy.Key duchy = 1;
+
+    // Resource key of the `DuchyCertificate` for this entry. Required.
+    // Immutable.
+    DuchyCertificate.Key duchy_certificate = 2;
```

Comment:
Can't this be inferred from Duchy.Key? (Duchy.duchy_certificate field of the Duchy)